mobile_config/src/lib.rs

```diff
@@ -94,3 +99,23 @@ impl std::fmt::Display for KeyRole {
         f.write_str(s)
     }
 }
+
+pub struct EpochPeriod {
+    pub period: Range<DateTime<Utc>>,
+}
+
+impl TryFrom<u64> for EpochPeriod {
+    type Error = anyhow::Error;
+
+    fn try_from(next_reward_epoch: u64) -> anyhow::Result<Self> {
+        let start_time = Utc
+            .timestamp_opt(0, 0)
+            .single()
+            .ok_or_else(|| anyhow!("Failed to get Unix epoch start time"))?
+            + Duration::days(next_reward_epoch as i64);
```

Comment:
chrono provides a unix constant.
```suggestion
       let start_time = DateTime::<Utc>::UNIX_EPOCH + Duration::days(next_reward_epoch as i64);
```

Then this can turn into a regular `From` implementation.

Reply:
nice, i had forgotten about that. 
addressed here: e8643f61bafa27c8c1a9680da8f625f068f01393